akvo/rsr/staticmap.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+"""Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the Akvo RSR module.
+For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+import collections
+from django.conf import settings
+
+
+Coordinate = collections.namedtuple('Coordinate', ['latitude', 'longitude'])
+
+Size = collections.namedtuple('Size', ['width', 'height'])
+
+
+class MapquestAdapter(object):
+
+    URL = "https://www.mapquestapi.com/staticmap/v5/map?key={}&locations={}&defaultMarker=via-lg-1890ff-6b72a8&zoom=8"
```

Comment:
When there are multiple lat/lng, hardcoding the zoom level doesn't seem to work correctly. For example, with these two co-ordinates `28.704059,77.10249||19.387389,85.051541` setting the zoom level to 8 makes both the points disappear from the map, and we zoom in at the mid-point of these two points. Instead not setting the zoom value or setting it to 6 makes the map appear correctly. 

I think the zoom level here needs to be dynamically figured out based on the number of locations and their distance from each other, or may be even just the number of points. The API doesn't seem to have a max-zoom, so if there is a single coordinate, it seems to be too zoomed in, in which case we could explicitly set a zoom level to 8.  

We could have some kind of a heuristic to calculate the dimensions of the bounding box, and set the zoom level accordingly.